src/features/payload-cms/converters/blog-article/index.tsx

```diff
@@ -30,12 +39,24 @@ export const BlogArticleConverter: React.FC<{
 
   const source = article.content.bannerImage.sizes?.large?.url ?? '';
   const altText = getImageAltInLocale(locale, article.content.bannerImage);
+  const formattedDate = formatBlogDate(article.content.releaseDate, locale);
+
   return (
     <>
       <article className="my-8 w-full max-w-2xl px-8 max-xl:mx-auto">
-        <HeadlineH1>{article.content.blogH1}</HeadlineH1>
+        {/* Back button */}
+        <div className="mb-6">
+          <LinkComponent
+            href={`/${locale}/blog`}
+            className="inline-flex items-center gap-2 text-red-600 transition-colors hover:text-red-700"
+          >
+            <ChevronsLeft className="h-4 w-4" />
+            {backToOverviewText[locale]}
+          </LinkComponent>
+        </div>
 
-        <div className="text-conveniat-green relative mt-10 aspect-[16/9] w-full text-lg">
+        {/* Banner image with correct 1:2 aspect ratio */}
+        <div className="text-conveniat-green relative mt-10 aspect-[1/2] w-full text-lg">
```

Comment:
this does not work.

<img width="385" height="806" alt="image" src="https://github.com/user-attachments/assets/96c2106f-f161-484f-8dd2-df10ba83b932" />